src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -190,5 +240,25 @@ public int Height
         ///    </para>
         /// </summary>
         public override string ToString() => "{Width=" + _width.ToString() + ", Height=" + _height.ToString() + "}";
```

Comment:
Minor nit - can use string interpolation instead of adding strings with + and ToString()